examples/original950.yaml

```diff
@@ -1,4 +1,7 @@
-- ladino: '"Ande se arrapa el guerko" es un lugar muy leshos.'
+- ladino: "Ande se arrapa el guerko" es un lugar muy leshos.
+  english: "Where the devil shaves" is a faraway place.
```

Comment:
Because the sentence starts with a quote, it needs to be inside another quote, a different type of quote. That's why there was the single-quote around this sentence. I'll fix it.